code/_onclick/item_attack.dm

```diff
@@ -56,8 +57,7 @@
 
 /obj/attacked_by(obj/item/I, mob/living/user)
 	if(I.force)
-		visible_message("<span class='danger'>[user] has hit [src] with [I]!</span>", null, null, 2, user)
-		//only witnesses close by and the victim see a hit message.
+		user.visible_message("<span class='danger'>[user] has hit [src] with [I]!</span>", "<span class='danger'>You hit [src] with [I]!</span>")

```

Comment:
All you needed to do was readd the 'hit with x' as the second argument, honestly.